proposals/user-scripts-api.md

```diff
@@ -133,6 +136,8 @@ where
 
 In the future, if we allow multiple user script worlds (see section in Future Work below), this method can be expanded to allow for a user script world identifier to customize a single user script world.
 
+The proposal at [multiple_user_script_worlds.md](multiple_user_script_worlds.md) expands the behavior of `userScripts.configureWorld`.
```

Comment:
I would move this to `Separate USER_SCRIPT Worlds`  on the `(Potential) Future Enhancements. That way it will be easier to find the multiple user scripts world proposal on this doc